src-plugins/itkFilters/itkMorphologicalFiltersProcessBase.cpp

```diff
@@ -70,6 +72,13 @@ void itkMorphologicalFiltersProcessBase::setParameter(double data, int channel)
     }
 }
 
+void itkMorphologicalFiltersProcessBase::setParameter(int data)
```

Comment:
To be honest I find the whole approach to ```setParameter``` confusing and badly designed. DTK tries to use the same mechanism for parameters and inputs (using channels). Parameters shouldn't be set using channels but through parameter names (so strings).

Anyways, I don't like any of the available approaches so why not.